src/timepicker/docs/readme.md

```diff
@@ -50,3 +50,7 @@ All settings can be provided as attributes in the `<uib-timepicker>` or globally
  * `max`
     _(Defaults: undefined)_ :
      Maximum time a user can select
+
+ * `timepicker-tabindex`
```

Comment:
I thought about it, but I had a concern...but I can't remember it now - I am willing to go that route though for simplicity.